### PR TITLE
In 146835671 release 2 hide screenings

### PR DESCRIPTION
**Pull request by NateOSI**

### CODE REVIEW

File: config/routes/inactive_release_one_and_two_constraint.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Routes
```
Comment (saraOSI) [inline config/routes/inactive_release_one_and_two_constraint.rb]: I think this is good for now, but (as per usual refactor guidelines) we should probably come up with a better constraint system when we have a third release.

File: spec/features/home_page_spec.rb [hunk 2]
```diff
@@ -42,15 +48,30 @@
...
     end
   end
 
+  scenario 'includes title and navigation links when release one is disabled' do
+    visit root_path
+    expect(page).to have_title 'Intake'
+    expect(page).to have_link 'Start Screening'
+  end
+
   scenario 'includes a list of saved screenings' do
```
Comment (AmanShurafa) [inline spec/features/home_page_spec.rb]: I would be happier if there is a context for this scenario, 'when no releases are enabled'

File: config/routes/inactive_release_one_and_two_constraint.rb (hunk 1 above)
Comment (AmanShurafa) [inline config/routes/inactive_release_one_and_two_constraint.rb]: +1 for looking for a better constraint system

File: app/assets/javascripts/components/HomePage.jsx [hunk 3]
```diff
@@ -47,7 +50,7 @@ export class HomePage extends React.Component {
           </ul>
         </div>
         <div className='col-md-9'>
-          <ScreeningsTable screenings={this.state.screenings} />
+          { IntakeConfig.isFeatureInactive('release_two') && <ScreeningsTable screenings={this.state.screenings} /> }
```
Comment (AmanShurafa) [inline app/assets/javascripts/components/HomePage.jsx]: Would prefer to have a helper method to handle the logic of why this table should or should not be shown.  Making the view know about features is brittle.

Result of this change ripple through remaining code...

Comment (saraOSI) [inline app/assets/javascripts/components/HomePage.jsx]: `isFeatureInactive` is already a helper method. What were you hoping to abstract further?

File: spec/features/home_page_spec.rb (hunk 2 above)
Comment (saraOSI) [inline spec/features/home_page_spec.rb]: We can add that.

File: spec/javascripts/components/screenings/ScreeningPageSpec.jsx [hunk 4]
```diff
@@ -512,6 +511,7 @@ describe('when referral_submit is OFF', () => {
     }
     const component = shallow(<ScreeningPage {...props} />)
     component.find('button[children="Submit"]').simulate('click')
+    expect(config.isFeatureActive).toHaveBeenCalledWith('referral_submit')
     expect(submitScreening).not.toHaveBeenCalledWith('99')
```
Comment (saraOSI) [inline spec/javascripts/components/screenings/ScreeningPageSpec.jsx]: Yep. Can fix that.

File: app/assets/javascripts/components/HomePage.jsx (hunk 3 above)
Comment (AmanShurafa) [inline app/assets/javascripts/components/HomePage.jsx]: We were hoping to abstract why the screenings table was not being shown.  As-is, the view is aware of the release two feature flag even though it is unaware of its specific implementation.

**Pull request closed without merging**